examples/src/cross-contract-call.js

```diff
@@ -10,7 +10,18 @@ class OnCall extends NearContract {
     @call
     set_person_on_call({ accountId }) {
         near.log(`Trying to set ${accountId} on-call`)
-        const status = near.jsvmCall('status-message.test.near', 'get_status', { account_id: accountId })
+        const promise = near.promiseBatchCreate('statusmessage.test.near')
+        near.promiseBatchActionFunctionCall(promise, 'get_status', bytes(JSON.stringify({ account_id: accountId })), 0, 30000000000000)
+        near.promiseThen(promise, near.currentAccountId(), '_set_person_on_call_private', bytes(JSON.stringify({ accountId: accountId })), 0, 30000000000000);
+    }
+
+    @call
+    _set_person_on_call_private({ accountId }) {
+        near.log(`_set_person_on_call_private called, accountId ${accountId}`)
+        if (near.currentAccountId() == !near.predecessorAccountId()) {
```

Comment:
maybe typo?
```suggestion
        if (near.currentAccountId() !== near.predecessorAccountId()) {
```

Reply:
My VSCode formatting works this way :/ But yes, I do not like it too.

Reply:
Looks like the VSCode formatting bug! `x == !y` is interpreted as `x == (!y)`, and it seems not what we want:
```
> 'a' == !'b'
false
> 'a' !== 'b'
true
```

Reply:
Wow, I was expecting the same behavior. Nice catch!